units/ftdc_rollups.go

```diff
@@ -149,5 +149,17 @@ func (j *ftdcRollupsJob) Run(ctx context.Context) {
 		if err != nil {
 			j.AddError(errors.Wrapf(err, "problem adding rollup %s for perf result %s", r.Name, j.PerfID))
 		}
+		j.markTimeSeriesAsUpdated(ctx, result, r)
 	}
 }
+
+func (j *ftdcRollupsJob) markTimeSeriesAsUpdated(ctx context.Context, result *model.PerformanceResult, rollup model.PerfRollupValue) {
+	id := model.TimeSeriesId{
```

Comment:
I've seen constructing an ID in a few functions now, is it possible to create a constrcutor for TimeSeriesID that takes the other model types?

Reply:
Refactored out I think.